src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -355,10 +355,7 @@ def handle_supabase_error(self):
 
     def to_db_cluster_status(self, data: ClusterStatus) -> Dict[str, Any]:
         db_cluster_status = data.dict()
-        db_cluster_status["updated_at"] = "now()"
-        if data.last_alert_at is not None:
-            db_cluster_status["last_alert_at"] = str(data.last_alert_at)
-            
+        db_cluster_status["updated_at"] = "now()"            
```

Comment:
We want to add the last_alert_at only if there's a value there, right?
So, in case the runner was restarted, but we haven't received an alert yet, we won't override the old value in the table.
Do we still follow that?

Reply:
If the robusta_sink is reconstructed when the runner is restarted,  Then it will send update_cluster_status in the constructor.  It will put null until first alert. (like install scenario)
but, other than that it will behave the same

I changed the type because i think the regular use case wont be needing the datetime methods and I'ts easier than converting it to string in many places.

Reply:
I think we don't want that. 
Because on every restart, we will get the "prometheus not connected" notification until the first alert.
I think we should not update that field, unless it has a value which isn't None